src/main/java/com/cloudbees/jenkins/support/SupportPlugin.java

```diff
@@ -396,6 +402,59 @@ public static void loadConfig() throws IOException {
             instance.load();
     }
 
+    private static final boolean logStartupPerformanceIssues = Boolean.getBoolean(SupportPlugin.class.getCanonicalName() + ".threadDumpStartup");
```

Comment:
@stephenc would it be easier if we piggybacked off of the environment variable: `-Djenkins.model.Jenkins.logStartupPerformance=true` instead of creating a custom one?